src/backend/logging/log_file.cpp

```diff
@@ -21,25 +21,29 @@ namespace logging {
 // Logger
 //===--------------------------------------------------------------------===//
 
-void LogFile::SetMaxCommitId(txn_id_t max_commit_id) {
-  max_commit_id_ = max_commit_id;
+void LogFile::SetMaxLogId(cid_t max_log_id_file) {
+  max_log_id_file_ = max_log_id_file;
 }
 
-txn_id_t LogFile::GetMaxCommitId() { return max_commit_id_; }
+cid_t LogFile::GetMaxLogId() { return max_log_id_file_; }
```

Comment:
It is better to declare those getter functions const.
And in my opinion, it is better to put all those getter and setter functions in header for clarity. But strictly separating implementation from declaration is also a good choice :).
The same goes with src/backend/logging/log_buffer.cpp file.